code/datums/outfits/spec_op.dm

```diff
@@ -181,11 +181,11 @@
 	name = "Kellion Security"
 	uniform = /obj/item/clothing/under/deadspace/ert/kellion
 	suit = /obj/item/clothing/suit/armor/vest/kellion
-	l_ear = null
+	l_ear = /obj/item/device/radio/headset/ert
 	mask = null
 	head = null
-	belt = null
-	back = /obj/item/weapon/storage/backpack/rucksack/green
+	belt = /obj/item/weapon/storage/belt/holster/security/tactical
+	back = /obj/item/weapon/rig/security
```

Comment:
unffortunately rigs don't function properly when placed into an outfit like this.

I'll fix it in future, but for now rigs need to be placed nearbby, in lockers or on tables

Reply:
I'll leave it as null, and placed in lockers instead.